### PR TITLE
fix(ChatHeaderContentView): Unreachable buttons in community header at min width

### PR DESCRIPTION
**Pull request by micieslak**
### What does the PR do

Instead of being unreachable under bell icon, icons are placed over chat name with some semi transparent background for readability.

Closes: #7647

### Affected areas

`ChatHeaderContentView`

### Screenshot of functionality (including design for comparison)

- [x] test changes in both light and dark theme?
- [x] I've checked the design and this PR matches it

[Kazam_screencast_00037.webm](https://user-images.githubusercontent.com/20650004/194289416-be90ae60-27e5-4db4-ba75-b6abdbd78190.webm)

### CODE REVIEW

**Review by caybro — APPROVED** [review]
Code looks good but I wonder if we shouldn't elide the channel title/subtitle instead... that's more of a design decision

**Review by alexandraB99 — APPROVED** [review]

**Review by noeliaSD — CHANGES_REQUESTED** [review]
I'm neither sure about this solution. I think it should be better to ask the design team before merging and closing it. 

WDYT @John-44, @benjthayer ?

**Comment by benjthayer** [comment]
Thanks @micieslak @noeliaSD it works well but I wonder if we should consider instead setting a min width for the main/central message pane panel so there is no overlap? As users of the app do we see the need for the app width to be reduced to that degree? @John-44

**Comment by micieslak** [comment]
Thank you @noeliaSD nad @benjthayer for your input! I agree that as a target we may want to have different solution here, but it requires further investigation/discussion and design. This PR just removes bugs causing that buttons are unreachable. So I think that it's still worth to merge it now to unblock it. In the future we can take a look once again on that and adjust to design when it's done for this case. Wdyt?

**Comment by benjthayer** [comment]
Sounds good to me @micieslak - your solution works really nicely!

**Review by noeliaSD — APPROVED** [review]
Ok. Could you add a new issue to give a follow up to this and really agree on the design of this? Thanks

**Comment by micieslak** [comment]
This PR will be merged as an interim fix. Follow up is defined in this ticket: https://github.com/status-im/status-desktop/issues/7862

**Pull request merged**
